packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -40,6 +41,23 @@ const findClosestValue = (value: number, divisibleBy: number) => {
     : upperValue * Math.sign(value);
 };
 
+/**
+ * Finds and swaps a provided key for it's right to left format.
+ */
+const rtlKey = (key: string) => {
```

Comment:
```suggestion
const rtlKey = (key: 'ArrowLeft' | 'ArrowRight') => {
```

As input is limited, I suggest to restrict it.

---

Another suggestion is to use `dir` as input:

```tsx
function flipArrowKeys(dir: 'ltr' | 'rtl', key: 'ArrowLeft' | 'ArrowRight') {
  if (dir === 'ltr') {
    return key
  }
}
```

In this case we will not to have memoized `getRTLSafeKey()`.

Reply:
I switched the method to this and moved it outside: 

```jsx
const getRTLSafeKey = (key: string, dir: 'ltr' | 'rtl') => {
  if (dir === 'rtl') {
    switch (key) {
      case 'ArrowLeft': {
        return 'ArrowRight';
      }

      case 'ArrowRight': {
        return 'ArrowLeft';
      }
    }
  }

  return key;
};
```

and in the keydown changed ev.key usage to use the `normalizedKey` instead:
```jsx
const normalizedKey = getRTLSafeKey(ev.key, dir);
```